docs/reference/jointparameters.md

```diff
@@ -82,7 +82,7 @@ When used for a rotational motion the value of `minStop` must be in the range [-
 When both `minStop` and `maxStop` are zero (the default), the hard limits are deactivated.
 The joint hard limits use ODE joint stops (for more information see the ODE documentation on `dParamLoStop` and `dParamHiStop`).
 
-Finally, note that when both soft (`minPosition` and `maxPosition`, see the [Motor](motor.md)'s "Motor Limits" section) and hard limits (`minStop` and `maxStop`) are activated, the range of the soft limits must be included in the range of the hard limits, such that `minStop <= minValue` and `maxStop>= maxValue`.
+Finally, note that when both soft (`minPosition` and `maxPosition`, see the [Motor](motor.md)'s "Motor Limits" section) and hard limits (`minStop` and `maxStop`) are activated, the range of the soft limits must be included in the range of the hard limits, such that `minStop <= minPosition` and `maxPosition <= maxStop`.
```

Comment:
It doesn't seem very intuitive to me to swap these inequalities, why not writing `maxStop >= maxPosition`? That seems clearer to me.

Reply:
For me, it was less clear because in the reading direction, you go through the first lower bound, the elements inside, and end up on the upper bound.

I know that mathematically it's exactly the same. As you prefer.

Reply:
@DavidMansolino: what do you think?

Reply:
I slightly prefer the ``maxStop >= maxPosition`` (the other made me think 2 sec about it before understanding, whereas this one is directly clear).